integrator/app/Services/GuzzleHttpRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Services;
+
+use GuzzleHttp\Client;
+
+class GuzzleHttpRequest
+{
+    protected $httpClient;
+
+    public function __construct(Client $httpClient)
+    {
+        $this->httpClient = $httpClient;
+    }
+
+    protected function get($url)
```

Comment:
You need add the [PHPDoc](https://docs.phpdoc.org/references/phpdoc/index.html) in **ALL** *functions/methods* like this:
![image](https://user-images.githubusercontent.com/34130446/66692212-8d25c380-ec72-11e9-99bd-02a734c4206b.png)